### PR TITLE
Update docs to provide information about the naming_convention argument

### PR DESCRIPTION
**Pull request by bigdatasourav**
# Example query results
<details>
  <summary>Results</summary>
  
```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -45,6 +45,58 @@ where
 
 ## Get started
 
+### Naming Convention
```
Comment (cbruno10) [inline docs/index.md:48]:
```suggestion
## Naming Convention
```

I'd also suggest moving this to below `## Custom Fields`, as it's pretty advanced and most users won't use it IMO.

File: docs/index.md [hunk 2]
```diff
@@ -45,6 +45,58 @@ where
...
+
+The naming_convention configuration option allows you to control the naming format for tables and columns in the plugin. This option influences how table and column names are formatted when querying Salesforce data through Steampipe. There are two supported values for the naming_convention option:
```
Comment (cbruno10) [inline docs/index.md:50]:
```suggestion
The `naming_convention` configuration argument allows you to control the naming format for tables and columns in the plugin. Below are the supported values:
```

File: docs/index.md [hunk 3]
```diff
@@ -45,6 +45,58 @@ where
...
+
+- api_native :
+  If you set naming_convention to api_native, the plugin will use the native format for table names, meaning there will be no salesforce_ prefix, and the table and column names will remain as they are in Salesforce.
```
Comment (cbruno10) [inline docs/index.md:53]:
```suggestion
### API Native

If `naming_convention` is set to `api_native`, the plugin will use the Salesforce naming conventions. Table and column names will have mixed case and table names will not start with `salesforce_`.

For example:

```sql
select
  "Id",
  "WhoCount",
  "WhatCount",
  "Subject",
  "IsAllDayEvent"
from
  "Event";
```

```
+---------------------+----------+-------------+---------+---------------+
| ID                  | WhoCount |  WhatCount  | Subject | IsAllDayEvent |
+----------------------------------------------+-------------------------+
| 00U2t0000000Mw3dEAD | 0        |  0          | test    | false         |
+---------------------+----------+-----------------------+---------------+
```

**Review by cbruno10 — CHANGES_REQUESTED** [review]
@bigdatasourav Please see suggestions, thanks!

File: docs/index.md (hunk 1 above)
Comment (bigdatasourav) [reply docs/index.md:48]: Updated

File: docs/index.md (hunk 2 above)
Comment (bigdatasourav) [reply docs/index.md:50]: updated

File: docs/index.md (hunk 3 above)
Comment (bigdatasourav) [reply docs/index.md:53]: updated

File: docs/tables/salesforce_account.md [hunk 4]
```diff
@@ -63,3 +67,37 @@ from
 where
   rating = 'Hot'
 ```
+
```
[the remark this answers is not in the document]
Comment (bigdatasourav) [reply docs/tables/salesforce_account.md:68]: updated

File: docs/tables/salesforce_account.md [hunk 5]
```diff
@@ -63,3 +67,37 @@ from
...
+### Show details about the turbot account
```
[the remark this answers is not in the document]
Comment (bigdatasourav) [reply docs/tables/salesforce_account.md:100]: updated

**Review by cbruno10 — APPROVED** [review]

**Comment by cbruno10** [comment]
Approved, but this PR should actually be part of https://github.com/turbot/steampipe-plugin-salesforce/pull/24, so my preference would be for the commits here to be cherry picked/moved into the other PR and then we can have a single commit with the feature + docs.

**Pull request merged**